libcaf_io/src/worker.cpp

```diff
@@ -31,11 +31,11 @@ namespace basp {
 // -- constructors, destructors, and assignment operators ----------------------
 
 worker::worker(worker_hub& hub, message_queue& queue, proxy_registry& proxies)
-    : next_(nullptr),
-      hub_(&hub),
-      queue_(&queue),
-      proxies_(&proxies),
-      system_(&proxies.system()) {
+  : next_(nullptr),
```

Comment:
Our [contributing](https://github.com/actor-framework/actor-framework/blob/master/CONTRIBUTING.md#breaking-statements) document says something else. There are a few other constructors that were formatted this way in this commit.